common/djangoapps/third_party_auth/api/tests/test_views.py

```diff
@@ -234,7 +239,7 @@ def test_user_mappings_with_query_params_multi_value_key(self, query_params, exp
         self._verify_response(response, expect_code, expect_data)
 
     def test_user_mappings_only_return_requested_idp_mapping_by_provider_id(self):
-        testshib2 = self.configure_saml_provider(name='TestShib2', enabled=True, idp_slug='testshib2')
+        testshib2 = self.configure_saml_provider(name='TestShib2', enabled=True, visible=True, idp_slug='testshib2')
```

Comment:
Sounds good. I kinda eventually went overboard to get the unit testing to play nice, but I can definitely pare things back a bit.